src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,45 @@
 package core.basesyntax;
 
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        Map<String, Integer> mapCsv;
+        try {
+            mapCsv = readingCsv(Path.of(fromFileName));
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from the file " + fromFileName, e);
+        }
+        List<String> result = new ArrayList<>();
+        result.add(SUPPLY + "," + mapCsv.get(SUPPLY));
+        result.add(BUY + "," + mapCsv.get(BUY));
+        result.add(RESULT + "," + (mapCsv.get(SUPPLY) - mapCsv.get(BUY)));
+        try {
+            Files.write(Path.of(toFileName), result);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to the file " + toFileName, e);
+        }
```

Comment:
looks a little bit messy, let's create three methods that will called in this one:
1. read from file, return List<String>
2. process List<String> and form report, return report
3. write to file